lib/connections/mongoose.js

```diff
@@ -26,9 +29,15 @@ const createConnection = () => {
   });
 };
 
+/** @type {Object.<string, Connection>} */
```

Comment:
I don't think you meant to put the dot here before the chevron.

Reply:
True 🙂